pkg/actions/addon/create.go

```diff
@@ -27,11 +27,11 @@ const (
 
 func (a *Manager) Create(addon *api.Addon, wait bool) error {
 	version := addon.Version
-	if addon.Version == "latest" {
+	if version == "latest" || !strings.Contains(version, "eksbuild") && version != "" {
```

Comment:
same here!

Reply:
perhaps worth adding a test for `1.2.3.eksbuild`?

Reply:
yup! awesome, thanks I'll put all this feedback in a commit!